cmd/cluster-init/update_jobs.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	prowconfig "k8s.io/test-infra/prow/config"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	IPFilePath      = "ci-operator/jobs/infra-periodics.yaml"
+	PrePostFilePath = "ci-operator/jobs/openshift/release"
```

Comment:
```go
api.Metadata{
    Org: "openshift",
    Repo: "release",
    Branch: "master",
}.RelativePath()
```

Reply:
I don't see any way to really make this work as is. It returns something like `openshift/release/openshift-release-master.yaml` which I would need to modify to be `openshift/release/openshift-release-master-presubmits.yaml`
I could create a new method on `api.Metadata` though if thats preferable to what I have.

Reply:
Oh, d'oh - that's the `ci-operator` file. I thought we had a similar one for job files.

Reply:
If you use the WriteToDir stuff though this is not needed anyway